### ISSUE TITLE
Document minimum resources for selfhosting

### ISSUE THREAD

**Issue by alexanderadam** [scale8/scale8-tag-manager-and-analytics#122]
**Is your feature request related to a problem? Please describe.**

Scale8 looks impressive. I'm currently setting up new infrastructure and now I'm curious how many resources each of the Scale8 services need and what specification you'd recommend.
And maybe there's even something to document on how to tweak a few things in case a lot of resources are needed (GitLab has a reference that looks like [this](https://docs.gitlab.com/omnibus/settings/memory_constrained_envs.html) in case this would help as inspiration).

**Describe the solution you'd like**
Some documentation on resources.

**Describe alternatives you've considered**
Not having any documentation regarding resources

**PS:** I'm currently building new infrastructure and I'm considering to try Scale8.
OpenID Connect seems to be the only missing feature and [I saw that you have it already on the roadmap](https://github.com/scale8/scale8-tag-manager-and-analytics/issues/104) which is great. :+1:

**Comment by cwbeck** [comment]
Thanks @alexanderadam for submitting this request.

May I ask what are you using Scale8 for? Tag Manager, analytics or both?

**Comment by alexanderadam** [comment]
Hi @cwbeck,
I'm using it not yet as I'm finishing the evaluation phase but I don't know _any_ modern proper GDPR compliant and selfhostable competitor for a solution that supports both so these are definitely strong features.

Thus I guess Scale8 might gain popularity if you're able to raise visibility.

**Comment by cwbeck** [comment]
Thanks @alexanderadam :)

**Comment by MyWay** [comment]
I agree. At least it would be good to share some data/example as a guideline.
